examples/inference/client.py

```diff
@@ -18,14 +18,36 @@ async def run_main(host: str, port: int, stream: bool = True):
     client = LlamaStackClient(
         base_url=f"http://{host}:{port}",
     )
+    try:
```

Comment:
you can avoid an explicit `try-catch` and just let the natural exception bubble up

Reply:
@ashwinb Thank you for your review! pushed the new changes!